web_src/js/semanticDropdown.js

```diff
@@ -852,10 +895,20 @@ $.fn.dropdown = function(parameters) {
                 callback();
               },
               onSuccess : function(response) {
+                var
+                  values          = response[fields.remoteValues]
+                ;
```

Comment:
This looks weird. Was this file copied that way?

Reply:
It's 

https://github.com/fomantic/Fomantic-UI/blob/2f3312f0f075e3a570dc96725be17e965b3b0964/dist/components/dropdown.js#L800-L802

Reply:
Presumably this file is the result of applying the patch from https://github.com/go-gitea/gitea/pull/8638#issuecomment-549175290 on to fomantic's dropdown.js

Reply:
Okay, that's fine if it is in their source.